pom.xml

```diff
@@ -70,7 +70,7 @@
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <java.version>1.8</java.version>
     <spring-boot.version>2.3.4.RELEASE</spring-boot.version>
-    <ff4j.version>1.8.9</ff4j.version>
+    <ff4j.version>1.8.10-SNAPSHOT</ff4j.version>
```

Comment:
We cannot have a SNAPSHOT version.

Reply:
Build [#528](https://travis-ci.com/github/ff4j/ff4j-spring-boot-starter-parent/builds/187980189) also failed with the following reason:

```
[ERROR] Failed to execute goal on project ff4j-spring-services: Could not resolve dependencies for project org.ff4j:ff4j-spring-services:jar:1.8.10-SNAPSHOT: The following artifacts could not be resolved: org.ff4j:ff4j-core:jar:1.8.10-SNAPSHOT, org.ff4j:ff4j-aop:jar:1.8.10-SNAPSHOT: Failure to find org.ff4j:ff4j-core:jar:1.8.10-SNAPSHOT in https://oss.sonatype.org/content/repositories/snapshots/ was cached in the local repository, resolution will not be reattempted until the update interval of sonatype-snapshots has elapsed or updates are forced 
```

Reply:
core ff4j 1.8.10 is on central now, let me bump to 1.8.10 and release 9closing this PR)